CMakeLists.txt

```diff
@@ -22,6 +22,8 @@ set(BRION_MAINTAINER
 set(BRION_LICENSE LGPL)
 
 set(COMMON_PROJECT_DOMAIN ch.epfl.bluebrain)
+set(MVD_TOOL_DISABLE_INSTALL TRUE) # enable MVD-tool embedded usage
+
```

Comment:
A superproject install always installs all artefacts. For packages, superproject builds are not used but you depend on packages.

If we want to address this, we'll need to do it in one place for all non-public subprojects.